bundle/libraries/libraries.go

```diff
@@ -92,13 +93,13 @@ func findArtifactsAndMarkForUpload(ctx context.Context, lib *compute.Library, b
 	}
 
 	if len(matches) == 0 && isLocalLibrary(lib) {
-		return fmt.Errorf("no library found for %s", libPath(lib))
+		return fmt.Errorf("file %s is referenced in libraries section but no such file found on local file system", libPath(lib))
 	}
 
 	for _, match := range matches {
 		af, err := findArtifactFileByLocalPath(match, b)
 		if err != nil {
-			cmdio.LogString(ctx, fmt.Sprintf("%s. Skipping %s. In order to use the library upload it manually", err.Error(), match))
+			cmdio.LogString(ctx, fmt.Sprintf("%s. Skipping uploading. In order to use the define 'artifacts' section", err.Error()))
 		} else {
```

Comment:
If #703 is merged then will this condition still be valid?

Reply:
@pietern it will be valid when an user defines and `artifact` section and then specifies libraries section which points to some different wheel. It might happen in case of monorepo where multiple wheel packages can be in different folder.

Reply:
Yeah that's right -- it still feels weird to have to define this section when referring to a file.

Let's keep this in the back of our head when going over these flows again.